packages/teams-js/src/public/runtime.ts

```diff
@@ -216,9 +303,18 @@ export function generateBackCompatRuntimeConfig(highestSupportedVersion: string)
 }
 
 const applyRuntimeConfigLogger = runtimeLogger.extend('applyRuntimeConfig');
-export function applyRuntimeConfig(runtimeConfig: IRuntime): void {
-  applyRuntimeConfigLogger('Applying runtime %o', runtimeConfig);
-  runtime = deepFreeze(runtimeConfig);
+export function applyRuntimeConfig(runtimeConfig: IBaseRuntime): void {
+  // Some hosts that have not adopted runtime versioning send a string for apiVersion, so we should handle those as v1 inputs
+  if (typeof runtimeConfig.apiVersion === 'string') {
+    runtimeConfig = {
```

Comment:
Probably would be good to use applyRuntimeConfigLogger to write a log statement inside of this if statement as well, since mobile hosts that do this might cause weirdness that we need to debug in the future.